meson.build

```diff
@@ -79,23 +79,14 @@ else
   pkg_prefix = option_pkg_prefix
 endif
 
-rootprefix = get_option('rootprefix')
-if rootprefix == ''
-rootprefix = '/'
-endif
-
-bindir = rootprefix / get_option('bindir')
-libdir = rootprefix / get_option('libdir')
+bindir = get_option('bindir')
+libdir = get_option('libdir')
 libexecdir = get_option('libexecdir')
-if os == 'Linux' and libexecdir == 'libexec'
-  libexecdir = 'lib'
-endif
-libexecdir = rootprefix / libexecdir
-rc_libexecdir = libexecdir / 'rc'
+sbindir = get_option('sbindir')
+rc_libexecdir = get_option('prefix') / libexecdir / 'rc'
 rc_bindir = rc_libexecdir / 'bin'
 rc_sbindir = rc_libexecdir / 'sbin'
 rc_shdir = rc_libexecdir / 'sh'
-sbindir = rootprefix / get_option('sbindir')
```

Comment:
sbindir got dropped

Reply:
It is defined lower in the file. I'm probably going to remove these variables later in a separate pr.

Reply:
oh, i missed it up there, apologies.